android/sdl_android/src/main/java/com/smartdevicelink/managers/lockscreen/LockScreenManager.java

```diff
@@ -232,6 +232,7 @@ public void onNotified(RPCNotification notification) {
 							msg.getUrl() != null) {
 						// send intent to activity to download icon from core
 						deviceIconUrl = msg.getUrl();
+						deviceIconUrl = deviceIconUrl.replaceFirst("http://", "https://");
```

Comment:
I see this is done in the icon caching PR so there is no need to make this change here

Reply:
Merged develop in and took the changes from the lockscreen caching